views/components/progress_bar.erb

```diff
@@ -0,0 +1,10 @@
+<% progress = [100, 100 * numerator / denominator].min %>
+<% color = (progress < 60) ? "bg-blue-500" : (progress < 80) ? "bg-yellow-500" : "bg-red-500" %>
+<div class="flex justify-between mb-1">
+  <span class="text-base font-medium text-blue-700 dark:text-white"><%= title %></span>
+  <span class="text-sm font-medium text-blue-700 dark:text-white"><%= numerator %>/<%= denominator %>
```

Comment:
Do we have a particular reason for making them blue? Blue gives the impression that they are clickable to me. I'd prefer gray, black, or orange instead. But it's totally personal

Reply:
Changed it to default text color.